.github/scripts/setup-env.sh

```diff
@@ -33,43 +32,15 @@ if [[ "${OS_TYPE}" == "macos" && $(uname -m) == x86_64 ]]; then
 fi
 echo '::endgroup::'
 
-echo '::group::Set PyTorch conda channel and wheel index'
-# TODO: Can we maybe have this as environment variable in the job template? For example, `IS_RELEASE`.
-if [[ (${GITHUB_EVENT_NAME} = 'pull_request' && (${GITHUB_BASE_REF} = 'release'*)) || (${GITHUB_REF} = 'refs/heads/release'*) ]]; then
-  CHANNEL_ID=test
-else
-  CHANNEL_ID=nightly
-fi
-PYTORCH_CONDA_CHANNEL=pytorch-"${CHANNEL_ID}"
-echo "PYTORCH_CONDA_CHANNEL=${PYTORCH_CONDA_CHANNEL}"
-
-case $GPU_ARCH_TYPE in
-  cpu)
-    GPU_ARCH_ID="cpu"
-    ;;
-  cuda)
-    VERSION_WITHOUT_DOT=$(echo "${GPU_ARCH_VERSION}" | sed 's/\.//')
-    GPU_ARCH_ID="cu${VERSION_WITHOUT_DOT}"
-    ;;
-  *)
-    echo "Unknown GPU_ARCH_TYPE=${GPU_ARCH_TYPE}"
-    exit 1
-    ;;
-esac
-PYTORCH_WHEEL_INDEX="https://download.pytorch.org/whl/${CHANNEL_ID}/${GPU_ARCH_ID}"
-echo "PYTORCH_WHEEL_INDEX=${PYTORCH_WHEEL_INDEX}"
-echo '::endgroup::'
-
 echo '::group::Create build environment'
 # See https://github.com/pytorch/vision/issues/7296 for ffmpeg
 conda create \
   --name ci \
   --quiet --yes \
   python="${PYTHON_VERSION}" pip \
-  ninja libpng jpeg \
-  'ffmpeg<4.3' \
-  -c "${PYTORCH_CONDA_CHANNEL}" \
-  -c defaults
```

Comment:
We currently don't install anything from the PyTorch conda channel and thus also don't need to specify it here.